### ISSUE TITLE
Improve the bootstrap process

### ISSUE THREAD

**Issue by philip-cc** [cyber-crypt-com/encryptonize-core#213]
**User story**
As an Encryptonize Admin
I want it to be easy to perform the inital Encryptonize setup
So that I don't have to implement custom deployment logic.

Currently it is only possible to bootstrap an admin user with create user scope by doing `docker/kubectl exec...`. This means that a deployer would additionally need a GRPC client to create any additional users. This can be cumbersome in a Kubernetes setup where another service in the cluster might need e.g READ/WRITE permissions. 

TODO: details

**Acceptance Criteria**
- ...

**Scope**
What is the scope of the change

**Not in Scope**
Anything that is not relevant for this change

**Non-functional requirements**
If any.. (performance, security, etc.)

**Additional context**

**Enabler stories**
- [ ] add any enabler story that should be done before starting this story to this list, and use the checkmark to track completion of enabler stories.
